Cargo.toml

```diff
@@ -1,3 +1,29 @@
 [workspace]
 resolver = "2"
 members = ["tss-esapi", "tss-esapi-sys"]
+
+[patch.crates-io]
```

Comment:
I wonder what's the state of the RustCrypto ecosystem now... do we still need this patch section? :thinking:

Reply:
Do we have a schedule for when they might release and update here? It'd be amazing to have this merged.

Reply:
Next couple of months I would say. I think all the new things we wanted to have in crypto-bigint are in now.